db/migrations/postgres/000046_create_oauth_apps.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS oauthapps (
+    id VARCHAR(26) PRIMARY KEY,
+    creatorid VARCHAR(26),
+    createat bigint,
+    updateat bigint,
+    clientsecret VARCHAR(128),
+    name VARCHAR(64),
+    description VARCHAR(512),
+    callbackurls VARCHAR(1024),
+    homepage VARCHAR(256)
+);
+
+CREATE INDEX IF NOT EXISTS idx_oauthapps_creator_id ON oauthapps (creatorid);
+
+ALTER TABLE oauthapps ADD COLUMN IF NOT EXISTS istrusted boolean DEFAULT false;
+ALTER TABLE oauthapps ADD COLUMN IF NOT EXISTS iconurl VARCHAR(512) DEFAULT '';
```

Comment:
In the current system we don't set defaults on creation and usually we prefer to avoid them. I am wondering if we should change these as well.

Reply:
Hmm, makes sense 👍